src/main.js

```diff
@@ -252,6 +252,11 @@ function configureCommandlineSwitchesSync(cliArgs) {
 	 */
 	app.commandLine.appendSwitch('disable-features', 'CalculateNativeWinOcclusion');
 
+	/**
+	 * Allow media to play even if no user gesture is used to fix issues like #176284
+	 */
+	app.commandLine.appendSwitch('--autoplay-policy', 'no-user-gesture-required');
```

Comment:
```suggestion
	app.commandLine.appendSwitch('autoplay-policy', 'no-user-gesture-required');
```

Reply:
I wonder why it worked with this change 🤔

Reply:
It would work with `--` as well, the cli parser in Chromium will ignore them https://source.chromium.org/chromium/chromium/src/+/main:base/command_line.cc;l=356-357. Suggested to remove them for consistency.